static/css/training.css

```diff
@@ -87,44 +89,64 @@ section.call-to-action .main-section .cta-image > img {
   border: 5px solid white;
 }
 
+body.cid-training #get-certified .col-nav {
+  flex-flow: column nowrap;
+  justify-content: space-between;
+}
+
+body.cid-training #get-certified .col-nav > * {
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex-basis: auto;
+}
+
+body.cid-training #get-certified .col-nav > br {
+  flex-grow: 1;
+  display: block;
+  min-height: 2em;
+}
+
+body.cid-training #get-certified .col-nav a.button {
+  flex: initial;
+  min-height: 1.5em;
+  text-align: center;
+  width: auto;
+  margin-left: auto;
+  margin-right: auto;
+}
+
 
 @media only screen and (max-width: 840px) {
-  section.call-to-action .main-section .cta-image.cta-image-before, section.call-to-action .main-section .cta-image.cta-image-after {
-    order: 0;
-  }
-  section.call-to-action .main-section .cta-image > img {
+  body.cid-training   section.call-to-action .main-section .cta-image > img {
     margin: 0;
   }
-  section.call-to-action .main-section .cta-image > img {
+  body.cid-training   section.call-to-action .main-section .cta-image > img {
     width: 7rem;
   }
-  section.call-to-action .main-section > div.call-to-action > div {
+  body.cid-training section.call-to-action .main-section > div.call-to-action > div {
     padding: 0 2rem 0 2rem;
   }
-  section.call-to-action .main-section .cta-text {
-    flex: 2 0 75vw;
-    max-width: initial;
-    padding: 5vw 0 0 0;
-  }
 }
 
 
+
 /*  GO FULL WIDTH AT LESS THAN 480 PIXELS */
 
 @media only screen and (max-width: 480px) {
-  .col { margin: 1% 0 1% 0%;}
-  .col-container { flex-direction: column; }
+  body.cid-training .col { margin: 1% 0 1% 0%;}
+  body.cid-training .col-container { flex-direction: column; }
+  body.cid-training #logo-cks { order: initial; }
 }
 
 @media only screen and (max-width: 650px) {
-  .col-nav {
+  body.cid-training .col-nav {
     display: block;
     width: 100%;
   }
-  .col-container { flex-direction: column; }
+  body.cid-training .col-container { flex-direction: column; }
 }
 
-.button{
+body.cid-training .buttoa n{
```

Comment:
?

Reply:
One typo. Generally, things look okay.
"Get Kubernetes Certified" text is not centered over the 3 go to certification panels.
In the future, you might want to link the information to the different icons, otherwise, the 3 icons
seem less meaningful.

Reply:
We could omit the icons. I'd welcome advice on laying out that page, including from people who've got graphic design experience more than I have.

Reply:
Typo fixed I think